test/components/numericAxisTests.ts

```diff
@@ -331,4 +331,40 @@ describe("NumericAxis", () => {
     });
     svg.remove();
   });
+
+  it("confines labels to the bounding box for the axis", () => {
+    var data = [
+      {x: "05/02/2014", y: 2},
+      {x: "05/03/2014", y: 3},
+      {x: "05/06/2014", y: 4}
+    ];
+    var SVG_WIDTH = 500;
+    var SVG_HEIGHT = 500;
+    var svg = generateSVG(SVG_WIDTH, SVG_HEIGHT);
+
+    var xScale = new Plottable.Scale.Time();
+    var yScale = new Plottable.Scale.Linear();
+    var linePlot = new Plottable.Plot.Line(xScale, yScale);
+    var xAxis = new Plottable.Axis.Numeric(xScale, "bottom");
+    var yAxis = new Plottable.Axis.Numeric(yScale, "left");
+
+    linePlot.attr("x", function(d: any) { return d3.time.format("%x").parse(d.x);}, xScale);
+    linePlot.project("y", "y", yScale);
+    linePlot.addDataset(data);
+
+    var chart = new Plottable.Component.Table([
+      [yAxis, linePlot],
+      [null, xAxis]
+    ]);
+    chart.renderTo(svg);
```

Comment:
We should be able to unit-test the `Axis` functionality without creating an entire chart. We can do this by setting a really wide `formatter` on the `Axis`, and rendering just the `Axis` to the `svg`.